ios/TransactionListView.swift

```diff
@@ -35,43 +35,6 @@ class TransactionListView: UIView, UITableViewDelegate, UITableViewDataSource {
     }
   }
   
-  func activelyWaitToPresentDiscoverSheetBack(controller: DiscoverSheetViewController) {
-    DispatchQueue.main.asyncAfter(deadline: .now() + 0.1) {
-      if (self.window?.rootViewController?.presentedViewController == nil) {
-        controller.moved = false;
-        controller.hacked = false;
-        self.window?.rootViewController?.present(controller, animated: true)
-      } else {
-        self.activelyWaitToPresentDiscoverSheetBack(controller: controller)
-      }
-    }
-  }
-  
-  @objc var avatarOptions: NSArray? {
-    didSet {
-      if #available(iOS 14.0, *), false {
-        let options: Array = avatarOptions as! Array<NSDictionary>;
-        let actions: [UIAction] = options.map { opt in
-          return UIAction(title: opt["label"] as! String, 
-          image: UIImage(systemName: opt["uiImage"] as! String), 
-          handler: { _ in 
-            self.avatarOnPressFunction(optionId: opt["id"] as! String)   
-          })
-        }
-        let contextMenu = UIMenu(title: "Profile Image", options: .displayInline, children: actions)
```

Comment:
very confused as to why this is here but 🤷‍♂️ 

just want to make sure its not affecting current functionality

Reply:
I share the confusion with you, let me link a line that ultimately decided that this piece of code was left unused and then because it was unused it was further refactored leaving us with what we have after Profiles feature.

Line 52
<img width="663" alt="image" src="https://user-images.githubusercontent.com/16062886/198227173-6c9c0803-04b1-4e67-881d-bc72cc71aee4.png">

This whole code was completely disabled because there was a false apart from `#available(...)` in the if statement. I was also confused as to whether this code is whether whether I can safely remove it. I enabled it and first of all it was crashing the app beacause the discover sheet was removed and there was some native code linked to it. But when I removed the discover sheet code it looked like this, I have never seen this in app, so I guess it was just dead. My thinking here is we should remove this native code since it is related to Transactions List that we will quite likely rewrite to FlashList or some other high performance list implementation.


<img width="428" alt="image" src="https://user-images.githubusercontent.com/16062886/198232109-b029ecd5-e581-495b-a75d-dbbaa2d25e9a.png">